src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  background-color: white;
+  margin: 0;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__header {
+  display: flex;
+  align-self: flex-end;
+  margin-right: 50px;
+}
+
+.ul__links {
+  margin: 0;
+  display: flex;
+}
+
+.nav__link {
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-left: 20px;
+  height: 60px;
+  text-align: center;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:hover::after {
+  content: "";
+  display: block;
+  background-color: #00acdc;
+  height: 4px;
+  border-radius: 8px;
+  position: relative;
+  top: -4px;
+}
+
+.is-active {
+  color: #00acdc;
```

Comment:
create a variable for this color, and you can reuse it after in all places you need

Reply:
And now use this variable here